src/traffic_server/InkIOCoreAPI.cc

```diff
@@ -161,6 +161,33 @@ TSThreadCreate(TSThreadFunc func, void *data)
   return reinterpret_cast<TSThread>(thread);
 }
 
+TSThread
+TSThreadCreateSignal(TSThreadFunc func, void *data, bool term_by_signal)
```

Comment:
I don't think I like this name, I thought was creating the _signal_, not the thread. `TSThreadCreateWithSignalling`?